airflow/providers/amazon/aws/executors/ecs/boto_schema.py

```diff
@@ -0,0 +1,101 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Schemas for easily and consistently parsing boto responses. Makes use of marshmellow for ORM like transformation of responses to python objects. The executor logic makes use of these.